src/include/settings/settings_manager.h

```diff
@@ -152,6 +152,9 @@ class SettingsManager {
   /** @return The ParamInfo corresponding to the given parameter; throws exception if doesn't exist. */
   const ParamInfo &GetParamInfo(const settings::Param &param) const;
 
+  /** @return The Param corresponding to the given name; throws exception if doesn't exist. */
```

Comment:
We did not expose this before, and I don't remember why. If anyone has objections?